addons/rating/static/tests/mock_server/models/rating_rating.js

```diff
@@ -0,0 +1,5 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class RatingRating extends models.ServerModel {
+    _name = "rating.rating";
+}
```

Comment:
And also this is in `im_livechat` in master, so gonna have to remove it from livechat and keep the one in `rating/`.

Reply:
Noted, thank you :+1: